vocdoni/bridge-ui
# Upgraded to dvote-js 1.11.4

# Short description
Upgraded to dvote-js 1.11.4 and added minor changes for `generateProof` in `lib/api.ts`

# How can it be tested
Create ERC20 proposal and check everything works. Before calling the oracle node, the last call to a web3 node is the `eth_getProof` method and not the `eth_getBlockByNumber`, because we are doing now the light check of the proof
